docs/_form-helpers/radio.md

```diff
@@ -61,8 +61,8 @@ Any other options not listed here will be applied to the input.
     <div class="form__group has-error form-radio">
         <label for="inputRadioLeft" class="control__label">Default radio</label>
         <div class="controls">
-            <input id="inputRadioLeft" name="inputRadioLeft" checked="" type="radio" class="form__control radio">
-            <span class="help-block is-error"><i class="icon-warning-sign"></i> Something went wrong</span>
+            <input id="inputRadioLeft" name="inputRadioLeft" checked="" type="radio" aria-describedby="guid-12345678" class="form__control radio">
+            <span class="help-block is-error" id="guid-12345678" role="alert"><i class="icon-warning-sign"></i> Something went wrong</span>
```

Comment:
SiteImprove may also want `aria-atomic` anywhere we use `role="alert"`

Reply:
Added